dcos_installer/upgrade.py

```diff
@@ -130,6 +130,7 @@
 
 if [[ "$SKIP_CHECKS" = "false" ]]; then
     T=300
+    set +e
```

Comment:
Let's use the long option form: `set +o errexit`. It's more readable, and it's consistent with the other uses of `set` in this script.

Reply:
done